pagopa/pn-frontend
# refactor: generate with openapi tool `infoPa` api + adapt api `getUserGroups()` into PA

## Short Description
Migrate the list of groups API to the BFF's API.

## List of Changes Proposed in This Pull Request
- Generated models from the BFF OpenAPI.
- Updated `getUserGroups()` with the BFF's API.
- Updated related tests.

## How to Test
Start PA and check if everything works as intended by going in the ApiKeys section and check for the lit of groups.
